### DISCUSSION TITLE
target values on the chart

### DISCUSSION THREAD

**Discussion by ukraine-sdg** [open-sdg/open-sdg#1471] [Q&A] [answered]
Is it possible to add target values to the charts? If yes, could you, please, help how to do this?

**Comment by brockfanning** [comment, marked as answer]
@ukraine-sdg There is an indicator configuration setting called "graph_target_lines" which is the easiest way to do this.

Note: In your case a few site configurations need to be corrected (as mentioned in another question) so first please [merge this PR](https://github.com/ukraine-sdg/site/pull/181), wait about 10 minutes, and then continue below:

Here are the steps for adding a graph target line:

1. Go to the indicator on your site
2. Click on the "Edit" tab
3. Click on "Edit Configuration"
4. Scroll down to "Graph target lines" and expand it
5. Click "+Target line" to add one
6. Leave "Series" and "Unit" blank, and leave "Label content" at its default value
7. Change "Value" according to what number you want the target line to appear on. In other words, the data value that you are trying to achieve.
7. At the top, click "Download configuration" to save the indicator configuration file
8. Click on "Go to repository"
9. Upload the configuration file
